modules/ROOT/pages/running-containers.adoc

```diff
@@ -1,7 +1,7 @@
 = Running Containers
 
 == Introduction
-Fedora CoreOS ships with both docker and podman installed. This page explains how to use systemd units to start and stop containers with podman.
+Fedora CoreOS ships with both https://www.docker.com[docker] and https://podman.io[podman] installed. This page explains how to use systemd units to start and stop containers with podman.
```

Comment:
It ships with `docker` but not Docker.  Specifically, it ships with [Moby](https://mobyproject.org/), Docker's upstream.

Reply:
Added a small insert explaining this.